pfrl/experiments/evaluator.py

```diff
@@ -515,6 +535,10 @@ def evaluate_and_update_max_score(self, t, episodes):
             self.max_score = mean
             if self.save_best_so_far_agent:
                 save_agent(self.agent, "best", self.outdir, self.logger)
+
+        if self.record_counter is not None:
+            self.record_counter += 1
+
         return mean
```

Comment:
Do you need to increment `record_counter`? Use current episode 
```
def evaluate_and_update_max_score(self, t, episodes)
```

Reply:
The values of `t` corresponds with the step. Actually, I think I will use `t` since it is more indicative of how far we are in the training process.